### PR TITLE
Fixing gemspec warnings on `gem build`

### PR DESCRIPTION
**Pull request by cleicar**

### CODE REVIEW

File: mercadopago.gemspec [hunk 1]
```diff
@@ -20,12 +20,12 @@ Gem::Specification.new do |s|
...
   # specify any dependencies here:
-  s.add_dependency 'json', '>= 1.4.6'
-  s.add_dependency 'faraday', '>= 0.9.0'
-  s.add_development_dependency 'pry'
-  s.add_development_dependency 'rake'
-  s.add_development_dependency 'byebug'
-  s.add_development_dependency 'test-unit'
-  s.add_development_dependency 'vcr'
-  s.add_development_dependency 'webmock'
+  s.add_dependency 'json', '~> 1.4', '>= 1.4.6'
```
Comment (mauro-oto) [inline mercadopago.gemspec]: We can remove the second part: `>= 1.4.6`. `~> 1.4` should already attempt to use the latest patch available.

File: mercadopago.gemspec [hunk 2]
```diff
@@ -20,12 +20,12 @@ Gem::Specification.new do |s|
...
+  s.add_dependency 'faraday', '~> 0.9.0'
+  s.add_development_dependency 'pry', '~> 0.11.1'
+  s.add_development_dependency 'rake', '~> 12.1', '>= 12.1.0'
```
Comment (mauro-oto) [inline mercadopago.gemspec]: Same for these, we can remove '>= 12.1.0` and just use `~>`.

**Comment by cleicar** [comment]
@mauro-oto the requested changes were applied

**Comment by mauro-oto** [comment]
@cleicar awesome, thanks! :+1:

**Pull request merged**